pkg/controllers/deployment/default_deployment.go

```diff
@@ -84,12 +85,36 @@ func getOAuthServerDeployment(
 	templateSpec.Volumes = append(templateSpec.Volumes, v...)
 	container.VolumeMounts = append(container.VolumeMounts, m...)
 
+	argsRaw, err := observeoauth.GetOAuthServerArgumentsRaw(&operatorConfig.Spec.ObservedConfig)
+	if err != nil {
+		return nil, fmt.Errorf("Unable to get audit sync data: %w", err)
```

Comment:
errors should start with lowercase letters

Reply:
Maybe lets create another PR with linter findings, wdyt?